Builders/WindowsBuilder.cs

```diff
@@ -32,8 +29,27 @@ public override Uploader CreateUploader()
                                + $" --icon=\"{installIcon}\""
                                + $" --noPortable";
 
-            if (!string.IsNullOrEmpty(Program.WindowsCodeSigningCertPath))
-                extraArgs += $" --signParams=\"/td sha256 /fd sha256 /f {Path.GetFullPath(Program.WindowsCodeSigningCertPath)} /p {codeSigningPassword} /tr http://timestamp.comodoca.com\"";
+            if (!string.IsNullOrEmpty(Program.WindowsCodeSigningMetadataPath))
+            {
+                string path = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile), ".nuget", "packages", "microsoft.trusted.signing.client");
+                const string dll_name = "Azure.CodeSigning.Dlib.dll";
+
+                string? dllPath = Directory.GetFiles(path, dll_name, SearchOption.AllDirectories).LastOrDefault(p => p.Contains("x64"));
+
+                if (dllPath == null)
+                    Logger.Error("Could not find path for Dlib.dll");
+
+                // We're using `signTemplate` here as we need to prefer the system `signtool.exe` in order for it to
+                // run azure code signing correctly on older windows versions.
+                //
+                // This can be changed back to `signParams` if velopack changes the signtool logic to fix this.
+                string signToolPath = Directory.GetDirectories(@"C:\Program Files (x86)\Windows Kits\10\bin", "*", SearchOption.AllDirectories)
```

Comment:
I'll just leave a comment for now because I've discovered you can also download the 22621 version of https://www.nuget.org/packages/Microsoft.Windows.SDK.BuildTools/ and do a similar look up to the above dlib one, for the signtool.

Probably not important though.